src/main/java/com/flightstats/hub/app/HubBindings.java

```diff
@@ -151,6 +160,32 @@ public static ObjectMapper objectMapper() {
         return mapper;
     }
 
+    @Provides
+    @Singleton
+    @Inject
+    public static MetricsConfig metricsReportersConfig(HubVersion hubVersion) {
```

Comment:
One final tweak, I think:
During our talk about naming things, you and Crozier brought up using `*Provider`s. I was really only used to putting providers in modules like this, but I like how you have the `InfluxdbReporterProvider`...Having `*ConfigProvider`s would let us test things with logic in them (like `clusterTag` here). Would you be up for moving this into a class that implements `Provider`?

Reply:
Absolutely, easy enough, and def testing advantage